ci-operator/jobs/infra-image-mirroring.yaml

```diff
@@ -959,10 +959,16 @@ periodics:
         failures=0
         for mapping in /etc/imagemirror/mapping_supplemental_ci_images_*; do
           echo "Running: oc image mirror -f=$mapping --skip-multiple-scopes"
-          if ! oc image mirror -f="$mapping" --skip-multiple-scopes; then
-            echo "ERROR: Failed to mirror images from $mapping"
-            failures=$((failures+1))
-          fi
+          attempts=3
+          for attempt in $( seq $attempts ); do
+            if ! oc image mirror -f="$mapping" --skip-multiple-scopes; then
```

Comment:
We need to break on success, not on failure

Reply:
...oops, good catch!